ldap2pg/manager.py

```diff
@@ -130,8 +135,7 @@ def inspect_ldap(self, syncmap):
             if 'ldap' in mapping:
                 logger.info(
                     "Querying LDAP %.24s... %.12s...",
-                    mapping['ldap']['base'],
-                    mapping['ldap']['filter'].replace('\n', ''))
+                    mapping['ldap']['base'], mapping['ldap']['filter'])
```

Comment:
Hmm, i removed explicitly `\n` from log message in f6f1f3d6 because it breaks grep and make logs ugly. I find it weird to have f6f1f3d6 reverted in another commit. Can you explain this change ?